arPullGlob.py

```diff
@@ -270,9 +239,11 @@ def CmStats():
     stDHi=[]
     
     pvList = makList()
-    codeFlt = ["0:FB_SUM", "0:RFS:INTLK_FIRST", "0:RFREADYFORBEAM"]
+#for when sonya fixes it
+#    codeFlt = ["FB_SUM", "RFS:INTLK_FIRST", "RFREADYFORBEAM"]
+    codeFlt = ["PHAFB_SUM", "RFS:INTLK_FIRST", "RFREADYFORBEAM"]
     for pv in pvList:
-        CavFltDat= (resultsIntlk[(str(pv)+ codeFlt[1])]['times'])
+        CavFltDat= (resultsIntlk[(str(pv)+ codeFlt[1])]['times']) #was ['times'] not [1]
```

Comment:
Comments above the line are preferable to comments after

Reply:
left over from debugger stage. I can delete.